llvm/lib/Target/AIE/AIEInterBlockScheduling.cpp

```diff
@@ -59,8 +59,8 @@ void emitBundlesInScoreboard(const std::vector<MachineBundle> &Bundles,
   const int AmountToEmit = std::min(TotalBundles, HR->getConflictHorizon());
   // Do not emit more than the specified by the conflict horizon. More
   // then this will not cause conflicts.
-  for (int i = TotalBundles - AmountToEmit; i < TotalBundles; i++) {
-    for (MachineInstr *MI : Bundles[i].getInstrs())
+  for (int I = TotalBundles - AmountToEmit; I < TotalBundles; I++) {
+    for (MachineInstr *MI : Bundles[I].getInstrs())
```

Comment:
This will conflict with https://github.com/Xilinx/llvm-aie/pull/163. Could you drop this commit for now?